plugins/callback/logstash.py

```diff
@@ -18,40 +18,79 @@
     options:
       server:
         description: Address of the Logstash server
-        env:
-          - name: LOGSTASH_SERVER
         ini:
           - section: callback_logstash
             key: server
-            version_added: 1.0.0
+        env:
+          - name: LOGSTASH_SERVER
         default: localhost
       port:
         description: Port on which logstash is listening
-        env:
-            - name: LOGSTASH_PORT
         ini:
           - section: callback_logstash
             key: port
-            version_added: 1.0.0
+        env:
+          - name: LOGSTASH_PORT
         default: 5000
+        version_added: 1.0.0
```

Comment:
This needs to be removed, and the `version_added: 1.0.0` above needs to be re-added. (This is at the wrong level.)